gemet/thesaurus/views.py

```diff
@@ -19,8 +19,10 @@
 from django_q.tasks import result
 
 from gemet.thesaurus.models import Concept, DefinitionSource, Group, Language
-from gemet.thesaurus.models import InspireTheme, Namespace, Property, SuperGroup
-from gemet.thesaurus.models import Term, Theme, Version, AsyncTask
+from gemet.thesaurus.models import (
+    InspireTheme, Namespace, Property, SuperGroup
```

Comment:
Why only reformat one line instead of all 3 imports from gemet.thesaurus.models?